server/backend/api/app/routers/router_user.py

```diff
@@ -0,0 +1,32 @@
+import logging
+from http import HTTPStatus
+
+from api.app import database
+from api.app.crud import crud_user
+from fastapi import APIRouter, Depends
+from sqlalchemy.orm import Session
+from api.app.schemas import FamUserUpdateResponse
+
+LOGGER = logging.getLogger(__name__)
+router = APIRouter()
+
+
+@router.post("", status_code=HTTPStatus.OK, response_model=FamUserUpdateResponse)
+def update_user_information_from_idim_source(
+    page: int = 1,
+    per_page: int = 100,
+    use_pagination: bool = False,
```

Comment:
I added the pagination option, but it's optional, we can choose to use it or not. And I'm not sure how to setup cronjob with pagination, but we can discuss that the update ticket.